Sage/carbon
# feat(button-toggle-group): surface fullWidth prop to ensure children flex to container width

fix #5490

### Proposed behaviour



adds fullWidth prop which ensures children `ButtonToggle` components flex to full width of parent container

![image](https://user-images.githubusercontent.com/114918852/204576949-6228f82e-c642-4921-a335-bb1da1644f47.png)


### Current behaviour



no support for flexing `ButtonToggle` children

### Checklist




#### QA


### Additional context



### Testing instructions



See new stories added

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
